tests/common/config_reload.py

```diff
@@ -16,13 +16,24 @@ def config_system_checks_passed(duthost):
         return False
 
     logging.info("Checking if Orchagent up for at least 2 min")
-    out = duthost.shell("systemctl show swss.service --property ActiveState --value")
-    if out["stdout"] != "active":
-        return False
+    if duthost.is_multi_asic:
+        for asic in duthost.asics:
+            out = duthost.shell("systemctl show swss@{}.service --property ActiveState --value".format(asic.asic_index))
+            if out["stdout"] != "active":
+                return False
+
+            out = duthost.shell(
+                "ps -o etimes -p $(systemctl show swss@{}.service --property ExecMainPID --value) | sed '1d'".format(asic.asic_index))
+            if int(out['stdout'].strip()) < 120:
+                return False
+    else:
+        out = duthost.shell("systemctl show swss.service --property ActiveState --value")
+        if out["stdout"] != "active":
+            return False
 
-    out = duthost.shell("ps -o etimes -p $(systemctl show swss.service --property ExecMainPID --value) | sed '1d'")
-    if int(out['stdout'].strip()) < 120:
-        return False
+        out = duthost.shell("ps -o etimes -p $(systemctl show swss.service --property ExecMainPID --value) | sed '1d'")
+        if int(out['stdout'].strip()) < 120:
+            return False
```

Comment:
Feel a little curious why is it dark green here? Does indentation use tab or 4 spaces?

Reply:
In the code we have spaces. Using vi, tried changing it to tabs, but git shows the file is the same and won't allow to commit.

I think the code is OK. It is just how the diff is displayed - the original code was left indented by 1 tab - that is possibly why.